bom/pom.xml

```diff
@@ -25,7 +25,9 @@
         <azure-sdk-bom.version>1.2.22</azure-sdk-bom.version>
         <azure.core.http.client.vertx.version>1.0.0-beta.17</azure.core.http.client.vertx.version>
         <!-- @sync com.azure:azure-identity:${azure-identity.version} dep:com.microsoft.azure:msal4j -->
-        <msal4j.version>1.15.0</msal4j.version>
+        <msal4j.version>1.14.3</msal4j.version>
```

Comment:
Good catch. Does it mean `msal4j.version` and `json-smart.verion` are determined by `${azure-identity.version}` which is implicitly determined by `azure-sdk-bom.version`? 

In other words, every time we update `azure-sdk-bom.version`, we should check the `azure-identity.versio` defined in the bom, and finally check if  `msal4j.version` and `json-smart.verion`  need to be updated or not.

Reply:
Yes, please keep azure-identity, msal4j, json-smart are compatible.